docs/mkdocs.yml

```diff
@@ -17,20 +17,20 @@ theme:
 extra:
     social:
         - icon: fontawesome/brands/github
-          link: https://github.com/mne-stools/mne-study-template
+          link: https://github.com/mne-tools/mne-bids-pipeline
```

Comment:
typo

Reply:
(incidentally, our Sphinx build infrastructure probably would have caught this because we have a weekly `linkcheck` run to make sure there aren't issues like this)

Reply:
except that there are a few links that always fail CIs, so some of us have learned to habitually ignore the linkcheck failures 🙄